app/src/main/java/com/amaze/filemanager/ui/fragments/preference_fragments/UiPrefsFragment.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2014-2021 Arpit Khurana <arpitkh96@gmail.com>, Vishal Nehra <vishalmeham2@gmail.com>,
+ * Emmanuel Messulam<emmanuelbendavid@gmail.com>, Raymond Lai <airwave209gt at gmail.com> and Contributors.
+ *
+ * This file is part of Amaze File Manager.
+ *
+ * Amaze File Manager is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.amaze.filemanager.ui.fragments.preference_fragments
+
+import android.os.Bundle
+import androidx.preference.Preference
+import com.afollestad.materialdialogs.MaterialDialog
+import com.amaze.filemanager.R
+
+class UiPrefsFragment : BasePrefsFragment() {
+    override val title = R.string.ui
+
+    private var dragAndDropPref: Preference? = null
```

Comment:
`lateinit var`

Reply:
Same as above: `findPreference` returns `Preference?`

Reply:
Makes sense, then leave it as is, and use "?" on every usage.